eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -202,8 +191,7 @@ abstract class SafeMojo extends AbstractMojo {
 
     /**
      * Used for object versioning implementation.
-     * If set to TRUE, external tojos are used instead of foreign ones and all
-     * inherited Mojos behave a bit differently.
+     * If set to TRUE all inherited Mojos behave a bit differently.
```

Comment:
@maxonfjvipon From this context it isn't clear  what is  "a bit differently", could you explain a bit more please? Moreover, do we still need the `withVersions` field?

Reply:
@volodya-lombrozo yes, we do. I'll extend the comment